starfish/core/intensity_table/test/test_intensity_table_coords.py

```diff
@@ -7,6 +7,7 @@
 
 from starfish.core.codebook.test.factories import codebook_array_factory
 from starfish.core.imagestack.test.factories import imagestack_with_coords_factory
+from starfish.core.intensity_table.test import factories
```

Comment:
This should be a relative import, i.e., `from .factories import synthetic_decoded_intenisty_table`